<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>669</width>
    <height>599</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:600; text-decoration: underline; color:#4387ca;&quot;&gt;Commercial 2013&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget_main">
      <property name="toolTip">
       <string/>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Catalogue</string>
       </attribute>
       <widget class="QTableView" name="tableView_in">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>20</y>
          <width>641</width>
          <height>371</height>
         </rect>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>420</y>
          <width>580</width>
          <height>31</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_Nouveau">
           <property name="text">
            <string>&amp;Nouveau</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_Supprimer">
           <property name="text">
            <string>&amp;Supprimer</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Commande</string>
       </attribute>
       <widget class="QTableView" name="tableView_Client">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>110</y>
          <width>641</width>
          <height>131</height>
         </rect>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>451</width>
          <height>111</height>
         </rect>
        </property>
        <property name="title">
         <string>Client</string>
        </property>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>0</y>
           <width>378</width>
           <height>108</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>#</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="spinBox"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Nom :</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_clientNom"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Pr√©nom :</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_clientPrenom"/>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLabel" name="label_9">
                <property name="text">
                 <string>Ville :</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_clientVille"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QLabel" name="label_10">
                <property name="text">
                 <string>Tel :</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_clientTel"/>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_nvClient">
              <property name="text">
               <string>Nouveau &amp;Client</string>
              </property>
              <property name="icon">
               <iconset resource="res.qrc">
                <normaloff>:/new/add/add.png</normaloff>:/new/add/add.png</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>480</x>
          <y>10</y>
          <width>162</width>
          <height>31</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_14">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Date :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="dateEdit"/>
         </item>
        </layout>
       </widget>
       <widget class="QTableWidget" name="tableWidget_Commande">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>300</y>
          <width>511</width>
          <height>131</height>
         </rect>
        </property>
        <column>
         <property name="text">
          <string># Produit</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Libelle</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Quantite</string>
         </property>
        </column>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>540</x>
          <y>320</y>
          <width>101</width>
          <height>116</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="pushButton_Valider">
           <property name="text">
            <string>&amp;Valider</string>
           </property>
           <property name="icon">
            <iconset resource="res.qrc">
             <normaloff>:/new/add/start.png</normaloff>:/new/add/start.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_Effacer">
           <property name="text">
            <string>&amp;Effacer</string>
           </property>
           <property name="icon">
            <iconset resource="res.qrc">
             <normaloff>:/new/add/remove.png</normaloff>:/new/add/remove.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_Annuler">
           <property name="text">
            <string>Annuler</string>
           </property>
           <property name="icon">
            <iconset resource="res.qrc">
             <normaloff>:/new/add/stop.png</normaloff>:/new/add/stop.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>250</y>
          <width>447</width>
          <height>36</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Produit</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_Produit"/>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Quantit√©</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinBox_Quantite"/>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_Ajouter">
           <property name="text">
            <string>&amp;Ajouter</string>
           </property>
           <property name="icon">
            <iconset resource="res.qrc">
             <normaloff>:/new/add/insert-object.png</normaloff>:/new/add/insert-object.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>669</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Fichier">
    <property name="title">
     <string>&amp;Fichier</string>
    </property>
    <addaction name="action_Quitter"/>
   </widget>
   <widget class="QMenu" name="menu_Evenement">
    <property name="title">
     <string>&amp;Evenement</string>
    </property>
    <addaction name="action_Gestion_du_catalogue"/>
    <addaction name="actionGestion_de_Commandes"/>
    <addaction name="separator"/>
    <addaction name="action_Cr_er_un_nouveau_client"/>
   </widget>
   <addaction name="menu_Fichier"/>
   <addaction name="menu_Evenement"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Nouvelle_commande">
   <property name="text">
    <string>&amp;Nouvelle commande</string>
   </property>
  </action>
  <action name="action_Quitter">
   <property name="text">
    <string>&amp;Quitter</string>
   </property>
  </action>
  <action name="action_Gestion_du_catalogue">
   <property name="text">
    <string>&amp;Gestion du Catalogue</string>
   </property>
  </action>
  <action name="actionGestion_de_Commandes">
   <property name="text">
    <string>Gestion de &amp;Commandes</string>
   </property>
  </action>
  <action name="action_Cr_er_un_nouveau_client">
   <property name="text">
    <string>&amp;Cr√©er un nouveau client</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lineEdit_clientNom</tabstop>
  <tabstop>lineEdit_clientPrenom</tabstop>
  <tabstop>lineEdit_clientVille</tabstop>
  <tabstop>lineEdit_clientTel</tabstop>
  <tabstop>pushButton_nvClient</tabstop>
  <tabstop>comboBox_Produit</tabstop>
  <tabstop>spinBox_Quantite</tabstop>
  <tabstop>pushButton_Ajouter</tabstop>
  <tabstop>pushButton_Valider</tabstop>
  <tabstop>pushButton_Effacer</tabstop>
  <tabstop>pushButton_Annuler</tabstop>
  <tabstop>tableWidget_Commande</tabstop>
  <tabstop>tableView_Client</tabstop>
  <tabstop>tabWidget_main</tabstop>
  <tabstop>spinBox</tabstop>
  <tabstop>dateEdit</tabstop>
  <tabstop>pushButton_Nouveau</tabstop>
  <tabstop>pushButton_Supprimer</tabstop>
  <tabstop>tableView_in</tabstop>
 </tabstops>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
